release_notes.md

```diff
@@ -1,3 +1,51 @@
+*******************************
+# Release Notes for Dymos 1.1.0
+
+July 22, 2021
+
+This is version 1.1.0 of Dymos.
+The release provides compatibility with OpenMDAO >3.10.0, updates the
+documentation to JupyterBook, and adds a few new features.
+
+## Backwards Incompatible API Changes & Deprecations
+
+* Removed vectorize_derivs option from phase objectives due to OpenMDAO update. [#605](https://github.com/OpenMDAO/dymos/pull/605)
+* Updated tags and changed the name of option `dynamic` to `static_target` on parameters.  [#591](https://github.com/OpenMDAO/dymos/pull/591)
```

Comment:
I think this should be reworded, since the behavior changed too (dynamic is the opposite of static_target).
"The `dynamic` argument on add_parameter has been removed. A new argument `static_target` has been added which has opposite meaning of `dynamic`.